packages/terra-hookshot/src/_HookshotUtils.js

```diff
@@ -183,6 +198,11 @@ const getBoundingRect = (boundingElement) => {
   return rect;
 };
 
+/**
+ * This function returns the attachment object for the target when using coordinate based positioning.
+ */
+const getCoordinateAttachment = () => ({ vertical: 'middle', horizontal: 'center' });
```

Comment:
Any reason not to rename this coordinateAttachment and make it an object vs. a function that returns a static object?

Reply:
good call